### PR TITLE
fix periodic sync yaml deployment, 404 handling

### PR DESCRIPTION
**Pull request by rmmh**

### CODE REVIEW

File: gubernator/github/periodic_sync.py [hunk 1]
```diff
@@ -60,6 +60,11 @@ def get_prs_from_github(token, repo):
     while True:
         logging.info('fetching %s', url)
         response = urlfetch.fetch(url, headers=headers)
+        if response.status_code == 404:
+            logging.warning('repo was deleted?')
```
Comment (fejta) [inline gubernator/github/periodic_sync.py]: which repo?

Comment (rmmh) [reply gubernator/github/periodic_sync.py]: it's logged earlier (in sync_repo)

**Pull request merged**